components/tidb_query/src/rpn_expr/types/test_util.rs

```diff
@@ -98,7 +99,19 @@ impl RpnFnScalarEvaluator {
                 ed
             })
             .collect();
-        let func = super::super::map_pb_sig_to_rpn_func(sig, &children_ed).unwrap();
+
+        // use validator_ptr to testing the test arguments.
+        let func: RpnFnMeta = super::super::map_pb_sig_to_rpn_func(sig, &children_ed).unwrap();
```

Comment:
Please merge the https://github.com/tikv/tikv/pull/5282 while it's merged into master.